### PR TITLE
Task/api 643 create static token handler plugin

### PR DESCRIPTION
**Pull request by dbrown-va**
JIRA Issue: https://vasdvp.atlassian.net/browse/API-643

This pull request introduces a new Kong plugin `health-apis-static-token-handler` that will be applied to the `/services/argonaut/v0/token` route along with reorganizing the repo to better allow for local development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,54 @@ Once the plugin is installed on the Kong instance, it can be configured via the
...
     "name": "health-apis-token-validator",
     "config": {
-        "verification_url": "{verification_url}",
-        "verification_timeout": {verification_timeout},
-        "api_key": "{api_key}"
+        "verification_url": "{verification-url}",
+        "verification_timeout": {verification-timeout},
+        "api_key": "{api-key}",
+        "static_token": "{static-token}",
+        "static_icn": "{static-icn}"
```
Comment (jgarnhart-va) [inline README.md:25]: Glad documentation isn't slipping by us. :)

File: kong/plugins/health-apis-static-token-handler/handler.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  ngx.status = status_code
+  ngx.header["Content-Type"] = "application/json"
+  ngx.say(message)
+  ngx.exit(status_code)
+  
+end
+
+
+HealthApisStaticTokenHandler.PRIORITY = 1010
```
Comment (jgarnhart-va) [inline kong/plugins/health-apis-static-token-handler/handler.lua:65]: ~Why 1010? Maybe a comment here.~ After talking with @dbrown-va, I think this value is fine.

File: README.md [hunk 3]
```diff
@@ -18,13 +18,54 @@ Once the plugin is installed on the Kong instance, it can be configured via the
...
+
+### Configuration
+
+Once the plugin is installed on the Kong instance, it can be configured via the Admin port.  Replace config entries with the correct values for your environment.
+
+```
+{
+    "name": "health-apis-token-validator",
+    "config": {
+        "static_refresh_token": "{static-refresh-token}",
```
Comment (jgarnhart-va) [inline README.md:50]: @patrickvinograd @edpaget - Just as our plugin that validates an incoming access token allows a special static value through, we now have this plugin to allow a static *refresh token* value through (which simply returns the static access token). 

This flow is used by Pluto in their testing harness. 

As such, we will need two more values added to credstash. I will DM @patrickvinograd on Slack providing values for:

- {static-refresh-token}
- {static-scopes}

**Review by jgarnhart-va — APPROVED** [review]

**Review by jjennings-va — APPROVED** [review]

**Comment by jgarnhart-va** [comment]
@patrickvinograd @edpaget - Once AdHoc team signs off on this new plugin of ours, we will need it deployed to the new `dev-api.va.gov/services/argonaut/v0/token` route Ed set up for us last week.

**Review by edpaget — APPROVED** [review]

**Pull request merged**